.github/actions/build-artifacts/cache/action.yml

```diff
@@ -9,4 +9,4 @@ runs:
       with:
         path: |
           ./packages/arb-token-bridge-ui/build
-        key: build-artifacts-${{ github.run_id }}-${{ github.run_attempt }}
+        key: build-artifacts-${{ github.run_id }}
```

Comment:
What's the reason behind this change?

Reply:
There were issues previously when rerunning failed tests, where this artifact would not exist. But I have reverted the change, and added a check for the file in the main workflow